src/lib/connectors/bookshelf/LicenceVersionPurposeConditionTypes.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const { bookshelf } = require('./bookshelf.js');
+
+module.exports = bookshelf.model('LicenceVersionPurposeConditionTypes', {
+  tableName: 'water.licence_version_purpose_condition_types',
+
+  idAttribute: 'licence_version_purpose_condition_type_id',
+
+  hasTimestamps: ['date_created', 'date_updated'],
+
+  licenceVersionPurposeConditionType () {
+    return this.belongsTo('LicenceVersionPurposeConditionType', 'licence_version_purpose_condition_type_id', 'licence_version_purpose_condition_type_id');
```

Comment:
I'm not certain this relationship is correct.  I wonder if it should be something like:
```
  licenceVersionPurposeConditions () {
    return this.belongsTo('LicenceVersionPurposeCondition', 'licence_version_purpose_condition_type_id', 'licence_version_purpose_condition_type_id');
```

Reply:
You're completely right. I must've copied/pasted and forgot to edit this line. Well spotted. 🙏